figures/scripts/figS2-tmb-compare-panels.R

```diff
@@ -178,7 +183,32 @@ ggsave(tcga_tmb_cdf_pdf, tmb_tcga_plot, width = 5, height = 6.15, # because labe
 useDingbats = FALSE)  
 
 
-
+# Export Zenodo CSV files --------------------
+
+# S2H
+tmb_pbta_plot_df %>%
+  # reorder columns and remove unnecessary columns, but keep hex since identity was plotted
+  dplyr::select(Kids_First_Biospecimen_ID, everything(),
+                -cancer_group, -cancer_group_abbreviation) %>%
+  # remove \n from wrapped display column
+  dplyr::mutate(cancer_group_display = stringr::str_replace(cancer_group_display, "\n", " ")) %>%
```

Comment:
excess display output
```suggestion
  # remove \n from wrapped display column
  dplyr::mutate(Primary_diagnosis = stringr::str_replace(Primary_diagnosis, "\n", " ")) %>%
  # remove "n = " from sample size column
  dplyr::mutate(sample_size = stringr::str_replace(sample_size, "n = ", "")) %>%
```